### PR TITLE
C3DC 754

### PR DESCRIPTION
**Pull request by Senait7-ux**

### CODE REVIEW

File: src/pages/inventory/tabs/TabsView.js [hunk 1]
```diff
@@ -19,9 +19,9 @@ const Tabs = (props) => {
     name: tab.name,
     hasToolTip: true,
     toolTipText: tab.toolTipText,
-    count: `(${props.dashboardStats[tab.count].toLocaleString()})`,
-    display: [tab.name, props.dashboardStats[tab.count].toLocaleString()],
-    clsName: `${tab.name}`.toLowerCase().replace(' ', '_'),
+    count: tab.count != "none" ? `(${props.dashboardStats[tab.count].toLocaleString()})` : "(NA)",
```
Comment (huuaho) [inline src/pages/inventory/tabs/TabsView.js:22]: The linter is throwing warnings for the use of '!=' and recommends changing it to '!==' (for both lines 22 and 23)

File: src/pages/inventory/tabs/DefaultTabTheme.js [hunk 2]
```diff
@@ -48,6 +48,17 @@ export const customTheme = {
           color: '#000000',
           borderTop: '6px solid #9664C7',
         },
+        '&.treatment': {
```
Comment (huuaho) [inline src/pages/inventory/tabs/DefaultTabTheme.js:51]: The tab colors needs to be updated to match the original ordering of the colors. (Blue, yellow, minty green, purple, dark green, burgundy)

File: src/bento/dashboardTabData.js [hunk 3]
```diff
@@ -910,8 +910,279 @@ export const GET_FILE_IDS_FROM_FILE_NAME = gql`
...
+        treatment_agent
+        dbgap_accession
+        __typename
+    }
+}`;
+export const GET_TREATMENT_RESPONSE_OVERVIEW_QUERY = gql`
+query treatmentResponseOverview(
+    # Demographics
+    $participant_ids: [String],
+    $ethnicity: [String],
```
Comment (huuaho) [inline src/bento/dashboardTabData.js:993]: treatmentResponseOverview and treatmentOverview queries need to be updated from the new BE changes

**Review by iksheth — APPROVED** [review]

File: src/bento/dashboardTabData.js [hunk 4]
```diff
@@ -1140,6 +1411,185 @@ export const tabContainers = [
...
+    ],
+    id: 'treatment_response_tab',
+    tabIndex: '4',
+    selectableRows: true,
+    tableDownloadCSV: customStudyTabDownloadCSV,
+    downloadFileName: 'C3DC Studies Download',
+    tableMsg: {
+      noMatch: 'No Matching Records Found',
+    },
+    addFilesRequestVariableKey: 'treatment_response_ids',
```
Comment (huuaho) [inline src/bento/dashboardTabData.js:1589]: these lines can be removed or commented out 1589-1591

**Comment by iksheth** [comment]
Merge it after #136

File: src/pages/inventory/tabs/DefaultTabTheme.js (hunk 2 above)
Comment (iksheth) [reply src/pages/inventory/tabs/DefaultTabTheme.js:51]: Resolved.

File: src/bento/dashboardTabData.js (hunk 4 above)
Comment (iksheth) [reply src/bento/dashboardTabData.js:1838]: Resolved.

File: src/bento/dashboardTabData.js [hunk 5]
```diff
@@ -1140,6 +1411,185 @@ export const tabContainers = [
...
+    ],
+    id: 'treatment_tab',
+    tabIndex: '4',
+    selectableRows: true,
+    tableDownloadCSV: customStudyTabDownloadCSV,
+    downloadFileName: 'C3DC Studies Download',
+    tableMsg: {
+      noMatch: 'No Matching Records Found',
+    },
+    addFilesRequestVariableKey: 'treatment_ids',
```
[the remark this answers is not in the document]
Comment (iksheth) [reply src/bento/dashboardTabData.js:1748]: Resolved.

File: src/bento/dashboardTabData.js (hunk 3 above)
Comment (iksheth) [reply src/bento/dashboardTabData.js]: Resolved.

File: src/pages/inventory/tabs/TabsView.js (hunk 1 above)
Comment (iksheth) [reply src/pages/inventory/tabs/TabsView.js:22]: We can keep it like this and next PR we can fix or make change to this one. Passing this for now.

**Pull request merged**